data/meta.json

```diff
@@ -0,0 +1 @@
+{"uuid":"4e872c12-2c8a-42b4-8c9d-d4d2880d7ca6"}
```

Comment:
The data directory should be ignored.

Reply:
Great. I don't remember if I forgot to add it back to `.gitignore` or if it was not there when I created a blank new beats project from the template. I mean: should it be added here? https://github.com/elastic/beats/blob/master/generator/beat/.gitignore

Reply:
Yes, it should probably be added here https://github.com/elastic/beats/blob/master/generator/beat/%7Bbeat%7D/.gitignore and inside the oder beat generator directories.

Reply:
There is also a log directory created, I wonder if we should add this one also there.